ProjetINFRA/src/test/java/fr/m2dl/infra/OrchestratorTest.java

```diff
@@ -0,0 +1,27 @@
+package fr.m2dl.infra;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class OrchestratorTest {
+
+	@Test
+	public void createOrchestratorTest() {
```

Comment:
Maybe we should rename it createOrchestratorWithAgent because you test more than juste the creation of an orchestrator.